keps/prod-readiness/sig-release/2572.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2572
```

Comment:
how does PRR (https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#production-readiness-review-questionnaire)  fit into this KEP it seems mostly N/A?

Reply:
I think it should not have one, as it doesn't directly have an impact like code changes would. I think this is a case of validation requirement, and not having separate validation for different KEP "types".

Reply:
gotcha! seems fair. :smile_cat: